From bc6a0f13ce40c99a634bf441425538d162d72f3c Mon Sep 17 00:00:00 2001
From: FaqZul <faqzul.developer@gmail.com>
Date: Tue, 20 May 2025 17:08:45 +0700
Subject: [PATCH] FreeDaah: frida:rpc

---
 Frida/Script.swift | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Frida/Script.swift b/Frida/Script.swift
index 9cd878d..48e9dbe 100644
--- a/Frida/Script.swift
+++ b/Frida/Script.swift
@@ -183,7 +183,7 @@ public final class Script: @unchecked Sendable, CustomStringConvertible, Equatab
             let payload = payloadAny as? [Any],
             payload.count >= 3,
             let kind = payload[0] as? String,
-            kind == "frida:rpc",
+            kind == "free:daah",
             let requestId = payload[1] as? Int,
             let status = payload[2] as? String
         else {
@@ -292,7 +292,7 @@ public final class Script: @unchecked Sendable, CustomStringConvertible, Equatab
     }
 
     private enum FridaRpcKind: String, Decodable, CustomStringConvertible {
-        case `default` = "frida:rpc"
+        case `default` = "free:daah"
 
         var description: String {
             return rawValue
-- 
2.47.3

